iceoryx_hoofs/source/log/platform_building_blocks/console_logger.cpp

```diff
@@ -116,8 +113,8 @@ void ConsoleLogger::createLogMessageHeader(const char* file,
 
     // NOLINTJUSTIFICATION snprintf required to populate char array so that it can be flushed in one piece
     // NOLINTNEXTLINE(cppcoreguidelines-pro-type-vararg,hicpp-vararg)
-    auto retVal = snprintf(&m_buffer[0],
-                           NULL_TERMINATED_BUFFER_SIZE,
+    auto retVal = snprintf(&getThreadLocalData().buffer[0],
+                           ThreadLocalData::NULL_TERMINATED_BUFFER_SIZE,
                            "\033[0;90m%s.%03d %s%s\033[m: ",
```

Comment:
Could you please define some `constexpr const char* COLOR_GRAY = "\033[0;90m";` and `constexpr const char* COLOR_RESET = "\033[m";` and use it here.
I think this piece is hard to understand for someone who is not fluent in terminal color codes.

Reply:
I smell an ASCII library :D

Reply:
Mmmhh I smell it too. Stop all your current work and let's implement `iceoryx_ascii`